### PR TITLE
tool: shorten 3 --help descriptions

### PR DESCRIPTION
**Pull request by bagder**
gen.pl already warned about these lines but we didn't listen

### CODE REVIEW

File: src/tool_help.c [hunk 1]
```diff
@@ -68,7 +69,7 @@ static const struct helptxt helptext[] = {
   {"    --cert-status",
    "Verify the status of the server certificate"},
   {"    --cert-type <type>",
-   "Certificate file type (DER/PEM/ENG)"},
+   "Certificate file type"},
```
Comment (Lekensteyn) [inline src/tool_help.c:72]: The line does not seem to be too long, maybe it is worth keeping the few available options?

File: src/tool_help.c [hunk 2]
```diff
@@ -391,19 +392,19 @@ static const struct helptxt helptext[] = {
...
   {"    --retry-connrefused",
    "Retry on connection refused (use with --retry)"},
   {"    --retry-delay <seconds>",
    "Wait time between retries"},
-  {"    --retry-all-errors",
-   "Retry all errors (use with --retry)"},
   {"    --retry-max-time <seconds>",
    "Retry only within this period"},
-  {"    --sasl-authzid <identity> ",
+  {"    --sasl-authzid",
```
Comment (Lekensteyn) [inline src/tool_help.c:407]: Ths option appears to accept an option, the manual page seems to be incomplete in this sense. Restore it and update the manual?
```c
      case '6': /* --sasl-authzid */
        GetStr(&config->sasl_authzid, nextarg);
        break;
```

File: src/tool_help.c [hunk 3]
```diff
@@ -467,8 +468,8 @@ static const struct helptxt helptext[] = {
    "Transfer based on a time condition"},
   {"    --tls-max <VERSION>",
    "Set maximum allowed TLS version"},
-  {"    --tls13-ciphers <list>",
-   "TLS 1.3 ciphersuites (OpenSSL)"},
+  {"    --tls13-ciphers <list of TLS 1.3 ciphersuites>",
```
Comment (Lekensteyn) [inline src/tool_help.c:471]: This seems pretty verbose and will probably go over the 79 column limit, what about just `<TLS 1.3 ciphersuites>` or `<ciphers>`?

Comment (bagder) [reply src/tool_help.c:471]: It doesn't go over the limit... `gen.pl` warns if it does.

File: src/tool_help.c (hunk 1 above)
Comment (bagder) [reply src/tool_help.c:72]: I didn't intentionally change this, as the commit message says. Someone must've edited the generated source previously and not the correct source file...

File: src/tool_help.c (hunk 2 above)
Comment (bagder) [reply src/tool_help.c:407]: Same here

**Comment by bagder** [comment]
Thanks a lot @Lekensteyn for your remarks and keen eyes. I've polished up the other ones too now.

**Pull request closed without merging**